Makefile.toml

```diff
@@ -153,8 +164,19 @@ script = '''
 
 is_portable = eq ${1} "--portable"
 
-targets = array llm_openai llm_anthropic llm_grok llm_openrouter llm_ollama
-for target in ${targets}
+# LLM targets
```

Comment:
This is the same code as the previous targets loop, please just add it there instead of copying and duplicating